troposphere/static/js/components/common/InstanceHistorySection.jsx

```diff
@@ -61,30 +126,25 @@ var InstanceHistorySection = React.createClass({
                 );
             }
         } else {
-            items = this.state.instanceHistory.map(function(historyItem) {
-                let formattedAUTotal = historyItem.get("total_hours"),
-                    formattedStartDate = moment(historyItem.get("start_date")).format("MMMM Do YYYY, h:mm a"),
-                    formattedEndDate = "Present";
-                if (historyItem.get("end_date") && historyItem.get("end_date").isValid()) {
-                    formattedEndDate = moment(historyItem.get("end_date")).format("MMMM Do YYYY, h:mm a");
-                }
-                return (<div key={historyItem.cid}>
-                            {historyItem.get("status")}:
-                            {formattedStartDate} -
-                            {formattedEndDate} -
-                            {formattedAUTotal}
-                        </div>);
-            });
             content = (
-                <ul>
-                    {deletedInfo}
-                    {items}
-                </ul>
+                <table className="clearfix table" style={{ tableLayout: "fixed" }}>
+                    <thead>
+                        <tr>
+                            <th style={{ width: "100px"}}>Status</th>
+                            <th>Start Date</th>
+                            <th>End Date</th>
+                            <th>Message</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {this.state.instanceHistory.map(this.renderHistoryRow) }
```

Comment:
You can define this as the stop of `render()` as: 
```
let { instanceHistory } = this.state;
```

Then we can remove the repeated `this.state`s in the conditional as well.